.circleci/config.yml

```diff
@@ -295,6 +295,26 @@ jobs:
           command: |
             if [[ $CIRCLE_PULL_REQUESTS ]]; then exit 0; fi;
             bash .scripts/deploy-assistify-chat.sh
+  
+  publish-docker:
+    <<: *defaults
+    docker:
+      - image: docker:17.05.0-ce-git
+    
+    steps:
+      - attach_workspace:
+          at: /tmp
+
+      - checkout
+
+      - run:
+          name: Build application Docker image
+          command: |
+            docker build -f .docker/Dockerfile.local .
+      - run:
+          command: docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}
+      - run:
+          command: docker push assistify/assistify-${BRANCH}:${CIRCLE_TAG}
```

Comment:
I think the image should be named `assistify/chat` with the tags
From branch

- develop => `develop`
- master => `latest`

If a tag has been specified, there should be an additional `assistify/chat:<tag>`